includes/class-reader-activation.php

```diff
@@ -257,6 +292,359 @@ public static function auth_cookie_expiration( $length, $user_id, $remember ) {
+	 */
+	public static function nav_menu_items( $output, $args = [] ) {
+		$args = (object) $args;
+
+		/** Do not alter items for authenticated non-readers */
+		if ( \is_user_logged_in() && ! self::is_user_reader( \wp_get_current_user() ) ) {
+			return $output;
+		}
+
+		/**
+		 * Menu locations to add the account menu items to.
+		 */
+		if ( ! in_array( $args->theme_location, self::ACCOUNT_LINK_MENU_LOCATIONS, true ) ) {
+			return $output;
+		}
+
+		$link = self::get_account_link();
+		if ( empty( $link ) ) {
+			return $output;
+		}
+
+		$item  = '<li class="menu-item">';
+		$item .= $link;
+		$item .= '</li>';
+
+		if ( empty( $output ) ) {
+			$menu_class = sprintf( '%s %s', $args->menu_class, self::get_element_class_name( 'account-menu' ) );
+			$output     = sprintf( $args->items_wrap ?? '<ul id="%1$s" class="%2$s">%3$s</ul>', $args->menu_id, $menu_class, $item );
+		} else {
+			$output = $output . $item;
+		}
+		return $output;
+	}
+
+	/**
+	 * Get the account icon SVG markup.
+	 *
+	 * @return string The account icon SVG markup.
+	 */
+	private static function get_account_icon() {
+		return '<svg aria-hidden="true" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M7.25 16.4371C6.16445 15.2755 5.5 13.7153 5.5 12C5.5 8.41015 8.41015 5.5 12 5.5C15.5899 5.5 18.5 8.41015 18.5 12C18.5 13.7153 17.8356 15.2755 16.75 16.4371V16C16.75 14.4812 15.5188 13.25 14 13.25L10 13.25C8.48122 13.25 7.25 14.4812 7.25 16V16.4371ZM8.75 17.6304C9.70606 18.1835 10.8161 18.5 12 18.5C13.1839 18.5 14.2939 18.1835 15.25 17.6304V16C15.25 15.3096 14.6904 14.75 14 14.75L10 14.75C9.30964 14.75 8.75 15.3096 8.75 16V17.6304ZM4 12C4 7.58172 7.58172 4 12 4C16.4183 4 20 7.58172 20 12C20 16.4183 16.4183 20 12 20C7.58172 20 4 16.4183 4 12ZM14 10C14 11.1046 13.1046 12 12 12C10.8954 12 10 11.1046 10 10C10 8.89543 10.8954 8 12 8C13.1046 8 14 8.89543 14 10Z" /></svg>';
+	}
+
+	/**
+	 * Get account link.
+	 *
+	 * @return string Account link HTML or empty string.
+	 */
+	private static function get_account_link() {
+		$account_url = '';
+		if ( function_exists( 'wc_get_account_endpoint_url' ) ) {
+			$account_url = \wc_get_account_endpoint_url( 'dashboard' );
+		}
+
+		/** Do not render link for authenticated readers if account page doesn't exist. */
+		if ( empty( $account_url ) && \is_user_logged_in() ) {
+			return '';
+		}
+
+		$class = function( ...$parts ) {
+			array_unshift( $parts, 'account-link' );
+			return self::get_element_class_name( $parts );
+		};
+
+		$labels = [
+			'signedin'  => \__( 'Account', 'newspack' ),
+			'signedout' => \__( 'Sign In', 'newspack' ),
+		];
+		$label  = \is_user_logged_in() ? 'signedin' : 'signedout';
+
+		$link .= '<a class="' . \esc_attr( $class() ) . '" data-labels="' . \esc_attr( htmlspecialchars( \wp_json_encode( $labels ), ENT_QUOTES, 'UTF-8' ) ) . '" href="' . \esc_url_raw( $account_url ?? '#' ) . '" data-newspack-reader-account-link>';
```

Comment:
`$link` should be first initialised:

```suggestion
		$link = '<a class="' . \esc_attr( $class() ) . '" data-labels="' . \esc_attr( htmlspecialchars( \wp_json_encode( $labels ), ENT_QUOTES, 'UTF-8' ) ) . '" href="' . \esc_url_raw( $account_url ?? '#' ) . '" data-newspack-reader-account-link>';
```

Reply:
Fixed in d6be56ecfbc22c6644acde24cc6140728fb8e3bb. Your suggestion was missing an extra space for linting